printctlr/controller.go

```diff
@@ -39,9 +39,9 @@ func (c Controller) ResourceAdded(r *unstructured.Unstructured) {
 // ResourceUpdated receives both an the old version and current version of a
 // custom resource and will print out the the custom resource was changed
 func (c Controller) ResourceUpdated(oldR, newR *unstructured.Unstructured) {
+	metrics.TotalEvents.Inc()
```

Comment:
Any particular reason for moving this to the top? Just wanting to understand.

Reply:
Fixed